src/engine/guidance/post_processing.cpp

```diff
@@ -821,6 +827,7 @@ std::vector<RouteStep> collapseTurns(std::vector<RouteStep> steps)
         else if (isCollapsableInstruction(current_step.maneuver.instruction) &&
                  current_step.maneuver.instruction.type != TurnType::Suppressed &&
                  steps[getPreviousNameIndex(step_index)].name_id == current_step.name_id &&
+                 // canCollapseAll is also checking for compatible turns
```

Comment:
I am referring to the function `compatible` by name. I can also say steps if that is less confusing.